test/memory/sungeun/refCount/arrays.chpl

```diff
@@ -141,5 +144,5 @@ proc main() {
   m2 = memoryUsed();
   writeln("\t", m2-m1, " bytes leaked");
   if printMemStats then printMemAllocs();
-
+  */
```

Comment:
I'd be curious to see if they still failed, but I think leaving a comment on why they failed at one point is sufficient if you don't feel up for it